src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,88 @@
+.nav {
+  width: 100%;
+  position: fixed;
+  top: 0;
+  z-index: 9999;
```

Comment:
Using this z-index like this will make debugging hard at a later point. You can make var for the index with different levels.